feature-libs/organization/administration/core/store/effects/b2b-user.effect.ts

```diff
@@ -629,12 +640,34 @@ export class B2BUserEffects {
     )
   );
 
+  // TODO: make b2BUserCreationNotifierService a required dependency
+  constructor(
+    actions$: Actions,
+    b2bUserConnector: B2BUserConnector,
+    routingService: RoutingService,
+    userAccountFacade: UserAccountFacade,
+    userIdService: UserIdService,
+    // eslint-disable-next-line @typescript-eslint/unified-signatures
+    b2BUserCreationNotifierService: B2BUserCreationNotifierService
+  );
+  /**
+   * @deprecated since 6.5
+   */
+  constructor(
+    actions$: Actions,
+    b2bUserConnector: B2BUserConnector,
+    routingService: RoutingService,
+    userAccountFacade: UserAccountFacade,
+    userIdService: UserIdService
+  );
   constructor(
     private actions$: Actions,
     private b2bUserConnector: B2BUserConnector,
     private routingService: RoutingService,
     private userAccountFacade: UserAccountFacade,
-    private userIdService: UserIdService
+    private userIdService: UserIdService,
+    @Optional()
+    private b2BUserCreationNotifierService?: B2BUserCreationNotifierService
```

Comment:
I think we could make this `b2BUserCreationNotifierService?` service as `protected`. This is recommended for the better extensibility.

Reply:
Changed